message/router.go

```diff
@@ -430,6 +461,31 @@ func (h *handler) run(middlewares []HandlerMiddleware) {
 	h.logger.Debug("Router handler stopped", nil)
 }
 
+type Handler struct {
+	router  *Router
+	handler *handler
+}
+
+func NewHandler(router *Router, handler *handler) *Handler {
```

Comment:
it probably may be private ;)

Reply:
or even we can create it without constructor (it's used only in the package scope once, so it's fine)